src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -56,11 +60,13 @@ const modalScreenListeners = {
 
 const propTypes = {
     network: PropTypes.shape({isOffline: PropTypes.bool}),
+    myPersonalDetails: PropTypes.objectOf(personalDetailsPropType),
```

Comment:
this prop is never passed in and will always be a default prop right? In that case, should we just modify it in componentDidMount?

Reply:
Ahh this is a good point, I think maybe this logic is better suited to `personalDetails.js`